### PR TITLE
Updates & Fixes

### PR DESCRIPTION
**Pull request by Grimsbain**
1) Replaced securecall('PlayerFrame_ShowVehicleTexture') and
securecall('PlayerFrame_HideVehicleTexture') with custom functions.
2) Added new function to move Shaman and Warlock totemframe depending on
certain conditions.
3) Updated DruidMana code.
4) Updated TotemFrame code.
5) Changed all frames to low strata. (Same as default UI frames.)
6) General formating fixes.

### CODE REVIEW

File: Interface/AddOns/oUF_Neav/core.lua [hunk 1]
```diff
@@ -211,7 +261,7 @@ local function PlayerToVehicleTexture(self, event, unit)
     self.RaidIcon:SetPoint('CENTER', self.Portrait, 'TOP', 0, -5)
     self.T[1]:SetPoint('BOTTOM', self.Name, 'TOP', 0, 8)
 
-    securecall('PlayerFrame_ShowVehicleTexture')
+    HideAltResources(self)
```
Comment (renstrom) [inline Interface/AddOns/oUF_Neav/core.lua]: Is `PlayerFrame_ShowVehicleTexture` gone, or is it just not sufficient anymore?

Comment (Grimsbain) [inline Interface/AddOns/oUF_Neav/core.lua]:
```
function PlayerFrame_ShowVehicleTexture()
	PlayerFrameVehicleTexture:Show();
	
	local _, class = UnitClass("player");	
	if ( PlayerFrame.classPowerBar ) then
		PlayerFrame.classPowerBar:Hide();
	elseif ( class == "SHAMAN" ) then
		TotemFrame:Hide();
	elseif ( class == "DRUID" ) then
		EclipseBarFrame:Hide();
	elseif ( class == "DEATHKNIGHT" ) then
		RuneFrame:Hide();
	elseif ( class == "PRIEST" ) then
		PriestBarFrame:Hide();
	end
end
```

Is the entire function and we don't need the `PlayerFrameVehicleTexture:Show()` call and the rest of it doesn't cover all of what we need.

**Comment by renstrom** [comment]
What's the custom orderhall bar?

**Comment by Grimsbain** [comment]
When you enter your order hall Blizzard has a bar that goes all they way across the screen. This just replaces it with a smaller custom version. It needs some polish (it is all in english) but it works.

Default: http://imgur.com/SrphkjI
Custom: http://imgur.com/gNAYxfZ

**Pull request merged**